docs/modules/widgets/pages/widgets.adoc

```diff
@@ -0,0 +1,89 @@
+= Widget: Compose Material
+
+This section exposes the custom UI Widget we've already developed so you don't have to reinvent the wheel.
+
+When using Compose, you are probably using Material to handle your theme.
+The downside is that there are two material libraries.
+
+- Material (default 2): `compose.material`
+- Material 3 : `compose.material3`
+
+[[install]]
+== Installing
+
+If you are using `material` and/or `material3`, you can add one or both dependency:
```

Comment:
Don't encourage adding both dependencies, that's an anti-pattern (that leads to freaking confusions).